v1.0/known-limitations.md

```diff
@@ -10,6 +10,8 @@ This page describes limitations we've identified in the [CockroachDB 1.0](../rel
 
 ## Removing all rows from large tables
 
+{{site.data.alerts.callout_info}}Resolved as of version 1.1. See <a href="https://github.com/cockroachdb/cockroach/pull/17016">#17016</a>.{{site.data.alerts.end}}
```

Comment:
It's only resolved for TRUNCATE. The problem persists for DELETE.

Reply:
Ugh. I didn't realize that. @vivekmenezes, are we likely to resolve this limitation on `DELETE` as well for 1.1?

Reply:
FYI it's tracked in https://github.com/cockroachdb/cockroach/issues/17921